### ISSUE TITLE
Problem with options when add item

### ISSUE THREAD

**Issue by memoadian** [Crinsane/LaravelShoppingcart#172]
Hi, I have a problem when add item in cart

```
		Cart::add([
			'id' => $id,
			'name' => $name,
			'qty' => $qty,
			'price' => $price,
			'options' => [
				'image' => $image,
				'sku' => $sku,
				'slug' => Cleaner::url($name)
			],
		]);
```

all variables, contains values, but when i get the content with Cart::content()

return this.

`{"948cb53c201bad627e7d0686e0932068":{"rowId":"948cb53c201bad627e7d0686e0932068","id":"1","name":"Pipa de agua recta","qty":"1","price":90,"tax":18.9,"subtotal":90}}`

JSON does not return options key.

I don't understand why.

Thank you

**Comment by memoadian** [comment]
Update, the problem appears only from version 2.0.2 or later.

**Comment by Crinsane** [comment]
Should be fixed by 0c3c859b6c8b55d6cd6be118cb96bc0c8f37063c
